src/libraries/System.Private.Xml/src/System/Xml/Schema/NamespaceList.cs

```diff
@@ -341,7 +341,6 @@ public static bool IsSubset(NamespaceList sub, NamespaceList super)
             }
             else if (o1._type == ListType.Set && o2._type == ListType.Set)
             { //clause 4
-                nslist = o1.Clone();
```

Comment:
Should it instead be the next three lines that are deleted?

Reply:
Yep

Reply:
Actually the logic is a bit different so kept this.